src/sass/initial/_layout.scss

```diff
@@ -81,8 +81,8 @@ body {
   @include config.above('sm-page-break') {
     display: grid;
     grid-template:
-      'bginput fginput' auto
-      'bgslide fgslide' auto / 1fr 1fr;
+      'bginput switch fginput' auto
+      'bgslide . fgslide' auto / 1fr min-content 1fr;
```

Comment:
I like having the button in the grid, but currently:

- The column gaps seem quite large, with only that button in between the colors
- The top (color form) and bottom (color formats/gamuts) grids no longer align

![Screenshot 2025-02-03 at 11 59 42 AM](https://github.com/user-attachments/assets/51087367-48ec-4e23-9930-ef60809501dd)